demos/src/main.mustache

```diff
@@ -1,477 +1,489 @@
-<header class="o-header o-header--light-on-dark js-header" data-trackable="header">
-	<div class="o-grid-container">
-		<div class="o-grid-row">
-			<div class="o-header__masthead" data-o-grid-colspan="12">
-					<div class="edition-switcher n-util-hide-no-js">
-						<button class="edition-switcher__button js-edition-switcher-button" data-trackable="edition-switcher-toggler">UK Edition</button>
-						<ul class="edition-switcher__editions" aria-hidden>
-							<li class="edition-switcher__edition">
-									<a href="/international?edition=international" class="edition-switcher__edition__link" data-trackable="international">International Edition</a>
-							</li>
-						</ul>
-					</div>
-				<div class="o-header__masthead__logo">
-					<a class="o-header__logo__link" data-trackable="logo-large" href="/" title="Go to Financial Times homepage">
-						<span class="n-util-visually-hidden">
+<header class="o-header o-header--light-on-dark" data-o-component="o-header">
```

Comment:
Future suggestion: store data separately and template it into the main.mustache to reduce repetition in the template.